### ISSUE TITLE
神奇的模型

### ISSUE THREAD

**Issue by newxhy** [ShineChen1024/MiaoBi#2]
那么它是否支持在流行的webui里，比如A1111、fooocus、forge或者comyfui使用？

**Comment by ShineChen1024** [comment]
of course, i will turn it into a full safetensor file when i have time, or you can do it yourself.

**Comment by caissonfiv** [comment]
期待safetensor版本

**Comment by newxhy** [comment]
期待safetensor版本

**Comment by newxhy** [comment]
今天来催更

**Comment by zmwv823** [comment]
通过抄袭其他插件和官方代码，做了一个玩具型的comfyui插件。webui集成度太高，非码农实在看不懂。
这个模型生图实现代码跟webui代码不一样，所以哪怕是转换后也没法使用中文文生图，只能自己抄代码改代码抄出一个插件来。
中文提示词关键在text_encoder和tokenizer这两个（就是CLIP），把这两传递进去，其他所有sd1.5模型都可以使用中文文生图。
另外，由于数据集限制，很多中文提示词是没法识别的。
![workflow](https://github.com/ShineChen1024/MiaoBi/assets/13308350/6c68d400-ea85-4747-8c07-2057bfddeb0a)
![image](https://github.com/ShineChen1024/MiaoBi/assets/13308350/518532f3-ab28-422a-af27-ec02d6e1e0f0)

**Comment by zmwv823** [comment]
网络环境特殊，safetensor模型传不动，还是用comfyui插件吧。

**Comment by zmwv823** [comment]
个人尝试转换成了safetensors，miaobi_beta0.9.safetensors，在目录内。
百度 https://pan.baidu.com/s/1gJoW02NlqIDhCBrDouP0SA?pwd=9999
夸克 https://pan.quark.cn/s/2991314ebd30
